app/models/edm/web_resource.rb

```diff
@@ -115,5 +117,10 @@ def media_blank?
     def europeana_supported_media_mime_type
       errors.add(:media, I18n.t('errors.messages.inclusion')) unless ALLOWED_CONTENT_TYPES.include?(media&.content_type)
     end
+
+    def queue_thumbnail
+      return unless media_changed?
+      ThumbnailJob.perform_later(id.to_s)
```

Comment:
Why `id.to_s` and not just `id`?

Reply:
mongoid returns the ids as Objects as opposed to just strings.
So it results in:
`Unsupported argument type: BSON::ObjectId`